lib/ash_json_api/controllers/schema.ex

```diff
@@ -14,6 +14,7 @@ defmodule AshJsonApi.Controllers.Schema do
       |> Jason.encode!()
 
     conn
+    |> Plug.Conn.put_resp_content_type("application/vnd.api+json")
```

Comment:
The json-api spec does not appear to specify an expected content type for a schema, I've naively assumed the schema content type should match the content type specified for all json api resources.